interfaces/builtin/desktop.go

```diff
@@ -288,6 +289,18 @@ func (iface *desktopInterface) MountConnectedPlug(spec *mount.Specification, plu
 		if !osutil.IsDirectory(dir) {
 			continue
 		}
+		if release.DistroLike("arch", "fedora") {
+			// XXX: on Arch and Fedora 32+ there is a known
```

Comment:
This should reference a bug so that the workaround can be removed OR so that the whole fragment can be changed not to share host cache at some point.

Reply:
Filed https://bugs.launchpad.net/snapd/+bug/1877109 I'll mention it in the comment.